src/turbomind/kernels/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ add_library(activation_kernels STATIC activation_kernels.cu)
 set_property(TARGET activation_kernels PROPERTY POSITION_INDEPENDENT_CODE  ON)
 set_property(TARGET activation_kernels PROPERTY CUDA_RESOLVE_DEVICE_SYMBOLS  ON)
 
+add_library(activation STATIC activation.cu)
```

Comment:
What's the relationship between "activation" and "activation_kernels"?
Additionally, is "activation_kernels" still used?